contrib/cirrus/lib.sh

```diff
@@ -203,6 +203,7 @@ in_podman() {
     for envname in $(awk 'BEGIN{for(v in ENVIRON) print v}' | \
                      egrep "$envrx" | \
                      egrep -v "CIRRUS_.+_MESSAGE" | \
+                     egrep -v "CIRRUS_.+_TITLE" | \
```

Comment:
OK, I'm a day late and dollar short -- I didn't even know buildah was using Cirrus -- but this makes me cringe. Do I understand correctly that the problem you're trying to address is that (something somewhere) can't handle envariables with spaces in the value? If so, this implementation is super fragile. May I recommend instead:
```bash
    if ! [[ $x =~ " " ]]; then    (add to list)
```
or, possibly even better, declare `envargs` as an array and pass it through that way:
```bash
    declare -a envargs=()
    ...
    envargs+=("-e" "$envname=$envvalue")
    ...
    podman run ... "${envargs[@]}"
```

Reply:
That is a more correct fix, I completely agree, but the one-line change we added to the master branch was smaller, and not being the person we'll call if/when it breaks, I'm reluctant to make big changes there.
My goal with the maintenance branches is to try to avoid diverging from later branches, so if we're going to do a more correct fix, it needs to land in master first.

Reply:
Point taken, thanks. But in case this isn't well known: the reason you are dealing with `CIRRUS_COMMIT_TITLE` today is that the fine folks at cirrus-ci added it yesterday, at my request. There is absolutely nothing preventing them from adding new envariables, with any values, at any time. This might perhaps merit a tracking issue.

Reply:
Agreed.